src/psyclone/tests/psyGen_test.py

```diff
@@ -2937,12 +2937,23 @@ def test_literal_gen_c_code():
 
 
 # Test BinaryOperation class
+def test_binaryoperation_initialization(capsys):
```

Comment:
pylint complains that `capsys` is unused. It also complains about the import of PSy on line 55 (as being unused).

Reply:
removed unused